.github/workflows/a-team.yml

```diff
@@ -8,6 +8,51 @@ on:
       - main
 
 jobs:
+  aws_infra_provisioning:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.ATEAM_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.ATEAM_AWS_SECRET_ACCESS_KEY }}
+      TF_VAR_ORG: osbuild
+      TF_VAR_REPO: automotive-ci
+      GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+
+    name: setup AWS resources
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Terraform Setup
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.0.2
+
+      - name: Terraform Format
+        id: fmt
+        run: terraform -chdir=tform/aws-resources/ fmt -check
+
+      - name: Terraform Init
+        id: init
+        run: terraform -chdir=tform/aws-resources/ init
+        continue-on-error: false
+
+      - name: Terraform Validate
+        id: validate
+        run: terraform -chdir=tform/aws-resources/ validate -no-color
+
+      - name: Terraform Plan
+        id: plan
+        run: terraform -chdir=tform/aws-resources/ plan -no-color -var='org=$TF_VAR_ORG' -var='repo=$TF_VAR_REPO'
+
+      - name: Terraform Apply
+        id: apply
+        run: terraform -chdir=tform/aws-resources/ apply -auto-approve -var='org=$TF_VAR_ORG' -var='repo=$TF_VAR_REPO'
+        continue-on-error: true
+
+      - name: Check Error
+        if: ${{ steps.apply.outputs.exitcode != 0 && !contains( steps.apply.outputs.stderr, 'BucketAlreadyOwnedByYou') }}
+        run: exit 1
+
   build:
     # Workflow can be invoked only when triggered from base repo. GitHub is
```

Comment:
Move this comment to the first occurrence

Reply:
addressed